### ISSUE TITLE
While parsing node number 26 [Transpose -> "96"]:

### ISSUE THREAD

**Issue by niuxiaozhang** [onnx/onnx-tensorrt#87]
![image](https://user-images.githubusercontent.com/44792435/49914649-17738e80-fecd-11e8-94f0-861acdaf6afe.png)

**Comment by niuxiaozhang** [comment]
----------------------------------------------------------------
Input filename:   /home/hadoop/projects/onnx_model/license_plate_recognition.onnx
ONNX IR version:  0.0.3
Opset version:    9
Producer name:    pytorch
Producer version: 0.4
Domain:           
Model version:    0
Doc string:       
----------------------------------------------------------------
While parsing node number 26 [Transpose -> "96"]:
ERROR: /home/erisuser/p4sw/sw/gpgpu/MachineLearning/DIT/release/5.0/parsers/onnxOpenSource/builtin_op_importers.cpp:1660 In function importTranspose:
[8] Assertion failed: perm.order[BATCH_DIM] == BATCH_DIM
ERROR: failed to parse onnx file

**Comment by zhenming33** [comment]
this may be caused by transpose batch dimension

**Comment by OverrideSkywalker** [comment]
老哥问题解决了吗

**Comment by persuelx** [comment]
Have you solved it？

**Comment by feitiandemiaomi** [comment]
@niuxiaozhang @persuelx @zhenming33  I meet the same problem ,how did you solve it?

**Comment by persuelx** [comment]
@feitiandemiaomi  sorry, I have not solverd it

**Comment by zhenming33** [comment]
@feitiandemiaomi @persuelx  in my case, i change shape from batch_size * channel * w to  w * batch_size * channel, don't transpose batch_size dimension in model

**Comment by feitiandemiaomi** [comment]
@zhenming33 你是对的，我看了源码，tensorrt确实不允许batch_size的维度发生变化，像我的代码out = out.permute(3, 0, 1, 2).view(out.size(3), out.size(0), -1)   改成out = out.permute(0, 3, 2, 1).view(out.size(3), out.size(0), -1) 即可以解决上述报错

**Comment by feitiandemiaomi** [comment]
@persuelx 可以借鉴

**Comment by persuelx** [comment]
@feitiandemiaomi  @zhenming33  thank u ，I have done it successfully

**Comment by feitiandemiaomi** [comment]
@persuelx can you paste the error code ,because I find the new problem.
![图片](https://user-images.githubusercontent.com/18325462/80074161-59a9e700-857b-11ea-8ad7-14af758c5957.png)

**Comment by persuelx** [comment]
My mistake is the same as the first floor(niuxiaozhang)，your mistake is not the same as mine。

**Comment by feitiandemiaomi** [comment]
@persuelx Does your code contain view or reshape , I want to verify my error

**Comment by persuelx** [comment]
en

**Comment by feitiandemiaomi** [comment]
def forward(self, x):
        out = self.features(x)
        out = self.avgpool(out)
        out = out.permute(3, 0, 1, 2).view(out.size(3), out.size(0), -1)
        out = self.classifier(out)
        out = F.log_softmax(out, dim=2)
        return out
Here are all the operations on shapes:
[1, 3, 32, 200] ->[1, 384, 4, 25] ->[1, 384, 1, 25] ->[25, 1, 384] ->[25, 1, 5990]

**Comment by feitiandemiaomi** [comment]
@persuelx  I can determine thar the error is view op.   can you share forward code?

**Comment by kevinch-nv** [comment]
Has anyone tried to repro this with later versions of TensorRT?

**Comment by kevinch-nv** [comment]
Closing due to inactivity - if this can be reproduced on the latest version of onnx-tensorrt feel free to open a new issue.
